### DISCUSSION TITLE
While using stream through WebRTC got this [RTSP source] dial tcp 192.168.0.14:554: connect: connection refused

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#3224] [Questions]
### Question

Here I am to get the rtsp of the ICRealtimne NVR and trying to convert it into webrtc I got this error 

[path mystream ] [RTSP source] dial tcp 192.168.0.14:554: i/o timeout
[path mystream ] [RTSP source] read tcp 192.168.0.32:46686->192.168.0.14:554: read: connection reset by peer
[path mystream ] [RTSP source] dial tcp 192.168.0.14:554: connect: connection refused

Where Can I find the all errors and their response like this error when this was the problem (this was not enabled, this is the process to do it like so).

I appreciate your help, Thank you.

**Comment by aler9** [comment]
Hello, the error

```
[path mystream ] [RTSP source] read tcp 192.168.0.32:46686->192.168.0.14:554: read: connection reset by peer
```

is a basic error that is returned by the operating system and it means that the TCP connection was closed somehow, either by the counterpart or because a timeout occurred.

Check that the NVR can be accessed by the machine in which you installed the server and that `192.168.0.14:554` is the right IP and port and that RTSP is the right protocol.

[a comment by an automated account is not shown]
